app/styles/modules/board.scss

```diff
@@ -8,128 +8,165 @@
   This contains the CSS related to the entire document
 */
 
-.board{
+.board {
+  .box {
+    .box-body {
+      .lane {
+        height: 100%;
+
+        .box {
+          height: 100%;
+          box-shadow: none;
+          
+          .box-body {
+            min-height: 45rem;
+            padding: 0rem;
+            border: 2px solid #fff;
+
+            .item {
+              cursor: move;
+              margin: 0.3rem;
+              padding: 0.1rem 0.5rem 0.1rem 1rem;
+              border: 0.1rem solid #eee;
+              border-left: 0.5rem solid #eee;
+              background-color: #fff;
+              border-right: 0.1rem solid #ccc;
+              border-top: 0.1rem solid #ccc;
+              border-bottom: 0.1rem solid #ccc;
+
+              .description {
+                word-wrap: break-word;
+                margin-bottom: 2rem;
+              }
+
+              p > img {
+                width: 10rem !important;
+                height: 10rem !important;
+              }
+
+              .img-circle {
+                width: 4rem;
+                right: 0;
+                position: absolute;
+                margin: -1.5rem 1rem;
+              }
+
+              .fa-ban {
+                color: #d33724;
```

Comment:
Lets handle it with mixins

Reply:
http://ember.projects4me/app/project/add12ff55acb-1902-6b9c-30d5411f7729/issue/3187